blocks/developers-toc/developers-toc.js

```diff
@@ -0,0 +1,130 @@
+import ComponentBase from '../../scripts/component-base.js';
+
+const sitePathPrefix = 'developers';
+
+export default class DeveloperToc extends ComponentBase {
+  ready() {
```

Comment:
Optionally for this type of component which needs to be initialized without having a block on the page, there is a feature used for breadcrumbs which could be reused. 

This will add the component in the first section on the page based on the bellow selector for `targetsSelectors`. 
Is it required to have it's own section?

Meta attribute `structure-${componentName}` -> `structure-developers-toc` set to true

Config required on the DeveloperToc class:
```
static loaderConfig = {
    ...ComponentBase.loaderConfig,
    targetsSelectors: 'main > div:first-child',
    targetsSelectorsLimit: 1,
  };
  
  extendConfig() {
    return [
      ...super.extendConfig(),
      {
        contentFromTargets: false,
        addToTargetMethod: 'replaceWith',
        targetsAsContainers: {
          addToTargetMethod: 'prepend',
          contentFromTargets: false,
        },
      },
    ];
  }
```